### DISCUSSION TITLE
I can't use Tailwind 3 with TypeScript

### DISCUSSION THREAD

**Discussion by omercsx** [tailwindlabs/tailwindcss#6315] [Help] [answered]
I am trying to use the new version of Tailwind with Typescript (I installed it with create-react-app@next which supports TS) however when I change anything it doesn't affect.

**Comment by simonswiss** [comment, marked as answer]
Hey!

A shot in the dark, but perhaps you have the wrong file extension in your [`content` files to watch](https://tailwindcss.com/docs/content-configuration) to generate the CSS (`.jsx` instead of `.tsx`)?

There are no reasons why Tailwind CSS wouldn't work in a TypeScript project.

**Reply by omercsx** [reply]
I changed content as `content: [
		'./src/**/*.{html,js}',
		'./src/**/*.{html,ts}',
		'./src/**/*.{html,tsx}',
	],` and its worked! Thank you!
